File: src/main/java/com/bot/db/mappers/VoiceChannelMapper.java
package com.bot.db.mappers;

import com.bot.models.InternalVoiceChannel;

import java.sql.ResultSet;
import java.sql.SQLException;

public class VoiceChannelMapper {

    public static InternalVoiceChannel mapSetToInternalVoiceChannel(ResultSet set) throws SQLException {
        return new InternalVoiceChannel(
                set.getString("id"),
                set.getString("guild"),
                set.getString("name"),
                set.getBoolean("voice_enabled")
        );
    }
}

File: src/main/java/com/bot/db/mappers/VoiceChannelMapper.kt
package com.bot.db.mappers

import com.bot.models.InternalVoiceChannel
import java.sql.ResultSet
import java.sql.SQLException

object VoiceChannelMapper {
    @JvmStatic
    @Throws(SQLException::class)
    fun mapSetToInternalVoiceChannel(set: ResultSet): InternalVoiceChannel {
        return InternalVoiceChannel(
                set.getString("id"),
                set.getString("guild"),
                set.getString("name"),
                set.getBoolean("voice_enabled")
        )
    }
}